care/users/api/serializers/user.py

```diff
@@ -197,7 +197,8 @@ def validate(self, attrs):
                 )
 
         if (
-            self.context["created_by"].user_type == User.TYPE_VALUE_MAP["Staff"]
+            self.context["created_by"].user_type
+            in (User.TYPE_VALUE_MAP["Staff"], User.TYPE_VALUE_MAP["Nurse"])
```

Comment:
It would be appropriate to just allow staff to create doctors

cc: @vigneshhari @gigincg

Reply:
I agree, but that would also mean, existing staffs (now Nurses) would lose permission from what they were able to do before (creating doctors). Is this fine? @gigincg

Reply:
@rithviknishad 

We need to make a call whether Staff/Nurse would have:
1. Administrative Access (Create Users/Doctors)
2. Patient Access


@aparnacoronasafe Can you add some clarity on this?

Reply:
@rithviknishad 

We need to make a call whether Staff/Nurse would have:
1. Administrative Access (Create Users/Doctors)
2. Patient Access


@aparnacoronasafe Can you add some clarity on this?

Reply:
Practically, I don't think we have "Staff" at every hospital today, to take care of Admin Activities. In this scenario this would become a District/State Admin Activity, if we go ahead and implement this.

Reply:
@aparnacoronasafe has shared an access control list. cc: @cp-Coder 

|Modules             |Sub-Modules         |Link                                       |Fields                                                                                                                                                                                                                                                                                                                                                 |Staff (Hospital admin)                                                         |Nurse                                                         |
|--------------------|--------------------|-------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------|--------------------------------------------------------------|
|Facility Management |Facility Details    |https://care.coronasafe.in/facility/1fabc5fb-d96d-455a-ab61-53e6e0483cbc/update|All                                                                                                                                                                                                                                                                                                                                                    |Edit                                                                           |Edit                                                          |
|                    |Bed Capacity        |https://care.coronasafe.in/facility/497fd4fc-c5c1-4ed3-9b57-64cc881087fe/bed|All                                                                                                                                                                                                                                                                                                                                                    |Edit                                                                           |Edit                                                          |
|                    |Doctor Capacity     |https://care.coronasafe.in/facility/497fd4fc-c5c1-4ed3-9b57-64cc881087fe/doctor|All                                                                                                                                                                                                                                                                                                                                                    |Edit                                                                           |Edit                                                          |
|                    |Corona Triage       |https://care.coronasafe.in/facility/497fd4fc-c5c1-4ed3-9b57-64cc881087fe/triage|All                                                                                                                                                                                                                                                                                                                                                    |Edit                                                                           |Edit                                                          |
|                    |Location/bed management|https://care.coronasafe.in/facility/497fd4fc-c5c1-4ed3-9b57-64cc881087fe/location|All                                                                                                                                                                                                                                                                                                                                                    |Edit                                                                           |Edit                                                          |
|                    |Delete Facility     |https://care.coronasafe.in/facility/512802b9-6f09-40c6-8ff3-9cffa24bc3ad|All                                                                                                                                                                                                                                                                                                                                                    |NO ACCESS                                                                      |NO ACCESS                                                     |
|Inventory Management|All                 |https://care.coronasafe.in/facility/497fd4fc-c5c1-4ed3-9b57-64cc881087fe/inventory|All                                                                                                                                                                                                                                                                                                                                                    |Edit                                                                           |Edit                                                          |
|Resource Management |Raising resource request|https://care.coronasafe.in/facility/497fd4fc-c5c1-4ed3-9b57-64cc881087fe/resource/new, https://care.coronasafe.in/resource |All                                                                                                                                                                                                                                                                                                                                                    |Edit                                                                           |Edit                                                          |
|Asset Management    |Registration and Updation|https://care.coronasafe.in/facility/497fd4fc-c5c1-4ed3-9b57-64cc881087fe/assets/new|All                                                                                                                                                                                                                                                                                                                                                    |Edit                                                                           |Edit                                                          |
|                    |asset configurations|                                           |                                                                                                                                                                                                                                                                                                                                                       |no access                                                                      |no access                                                     |
|Sample Testing      |Sample test request generation|https://care.coronasafe.in/facility/a31e6a24-0146-4043-b295-3a6c2fe7e095/patient/a709c84e-28e8-41ee-bf5e-f7beb7e859f3/sample-test|All                                                                                                                                                                                                                                                                                                                                                    |NO ACCESS                                                                      |Edit                                                          |
|                    |Sample test management|https://care.coronasafe.in/sample          |All                                                                                                                                                                                                                                                                                                                                                    |NO ACCESS                                                                      |NO ACCESS                                                     |
|Shifting            |Shifting request generation|https://care.coronasafe.in/facility/1fabc5fb-d96d-455a-ab61-53e6e0483cbc/patient/dda0caa4-a52c-49cd-9dd3-7fe8093aaa76/shift/new|All                                                                                                                                                                                                                                                                                                                                                    |NO ACCESS                                                                      |Edit                                                          |
|Patient Management  |Patient Registration and updation |https://care.coronasafe.in/facility/015ad911-39b7-4104-b304-bf8178c33830/patient/76742faa-30dd-4eb9-9649-fe718f947ec0/update|All                                                                                                                                                                                                                                                                                                                                                    |Only view access to https://care.coronasafe.in/patients                        |Edit                                                          |
|                    |Disable/Enable Transfer|https://care.coronasafe.in/facility/015ad911-39b7-4104-b304-bf8178c33830/patient/76742faa-30dd-4eb9-9649-fe718f947ec0|-                                                                                                                                                                                                                                                                                                                                                      |NO ACCESS                                                                      |Edit                                                          |
|                    |Consultation        |https://care.coronasafe.in/facility/015ad911-39b7-4104-b304-bf8178c33830/patient/76742faa-30dd-4eb9-9649-fe718f947ec0/consultation/2ad89b7c-7bff-4035-a3c6-36d53551a8f0/update|All                                                                                                                                                                                                                                                                                                                                                    |NO ACCESS                                                                      |Edit                                                          |
|                    |Log Updates         |https://care.coronasafe.in/facility/a31e6a24-0146-4043-b295-3a6c2fe7e095/patient/a709c84e-28e8-41ee-bf5e-f7beb7e859f3/consultation/2d6cb153-19e8-4339-8796-3c71c44cbd68/daily-rounds|All                                                                                                                                                                                                                                                                                                                                                    |NO ACCESS                                                                      |Edit                                                          |
|                    |Update Bed          |https://care.coronasafe.in/facility/015ad911-39b7-4104-b304-bf8178c33830/patient/76742faa-30dd-4eb9-9649-fe718f947ec0/consultation/2ad89b7c-7bff-4035-a3c6-36d53551a8f0/update, https://care.coronasafe.in/facility/015ad911-39b7-4104-b304-bf8178c33830/patient/76742faa-30dd-4eb9-9649-fe718f947ec0/consultation/2ad89b7c-7bff-4035-a3c6-36d53551a8f0|Update bed in consultation form and the patient dashboard                                                                                                                                                                                                                                                                                              |NO ACCESS                                                                      |Edit                                                          |
|                    |Logging Investigation|https://care.coronasafe.in/facility/a31e6a24-0146-4043-b295-3a6c2fe7e095/patient/a709c84e-28e8-41ee-bf5e-f7beb7e859f3/consultation/2d6cb153-19e8-4339-8796-3c71c44cbd68/investigation/|All                                                                                                                                                                                                                                                                                                                                                    |NO ACCESS                                                                      |Edit                                                          |
|                    |Investigation summary generation|https://care.coronasafe.in/patient/76742faa-30dd-4eb9-9649-fe718f947ec0/investigation_reports|Generating the investigation summary to view all existing investigation reports                                                                                                                                                                                                                                                                        |NO ACCESS                                                                      |Edit                                                          |
|                    |patient files (including audio recording)|https://care.coronasafe.in/facility/015ad911-39b7-4104-b304-bf8178c33830/patient/76742faa-30dd-4eb9-9649-fe718f947ec0/files/|Both file upload and voice recording                                                                                                                                                                                                                                                                                                                   |NO ACCESS                                                                      |Edit                                                          |
|                    |Discharge a patient |                                           |                                                                                                                                                                                                                                                                                                                                                       |NO ACCESS                                                                      |Edit                                                          |
|User                |Add User            |                                           |                                                                                                                                                                                                                                                                                                                                                       |Create users for Staff, Doctors, nurses, pharmasist, volunteer                 |Create users for Staff, Doctors, nurses, pharmasist, volunteer|
|                    |Delete User         |                                           |                                                                                                                                                                                                                                                                                                                                                       |NO ACCESS                                                                      |NO ACCESS                                                     |
|Notice board        |Notifying hospitals |                                           |                                                                                                                                                                                                                                                                                                                                                       |NO ACCESS                                                                      |NO ACCESS                                                     |
|                    |Receiving notification through notice board|                                           |                                                                                                                                                                                                                                                                                                                                                       |Yes                                                                            |Yes                                                           |
|Notifications       |What notifications are received|                                           |                                                                                                                                                                                                                                                                                                                                                       |Only Notice                                                                    |All                                                           |
|External Result     |Uploading external result|                                           |                                                                                                                                                                                                                                                                                                                                                       |NO ACCESS                                                                      |Edit                                                          |
|                    |Making patient card from external result|                                           |                                                                                                                                                                                                                                                                                                                                                       |NO ACCESS                                                                      |Edit                                                          |

Reply:
@aparnacoronasafe has shared an access control list. cc: @cp-Coder 

I've attached it as a comment in the issue itself: https://github.com/coronasafe/care_fe/issues/6240#issuecomment-1783827403

Reply:
Okay, I will update the PR according to the access list that was shared